utility/infrastructure/models.py

```diff
@@ -0,0 +1,27 @@
+from datetime import datetime
+
+from sqlalchemy import Column, VARCHAR, Integer, ForeignKey, UniqueConstraint, null, DECIMAL, BIGINT, Index, TEXT
+from sqlalchemy.dialects.mysql import JSON, TIMESTAMP, TINYINT, BIT
+from sqlalchemy.ext.declarative import declarative_base
+from sqlalchemy.orm import relationship
+
+Base = declarative_base()
+
+class HistoricalCryptoFiatExchangeRates(Base):
+    __tablename__ = "historical_crypto_fiat_exhange_rates"
+    id = Column("id", Integer, primary_key=True, autoincrement=True)
+    crypto_symbol = Column("crypto_symbol", VARCHAR(100), nullable=False)
+    fiat_symbol = Column("fiat_symbol", VARCHAR(100), nullable=False)
+    crypto_rate = Column("crypto_rate", DECIMAL(19, 8), nullable=False)
+    fiat_rate = Column("fiat_rate", DECIMAL(19, 8), nullable=False)
+    added_at = Column("added_at", TIMESTAMP(timezone=False), nullable=True, default=datetime.utcnow(), index=True)
```

Comment:
please change added_at to created_at so that we have the same naming convention across all the services

Reply:
Updated to `created_at` from `added_at`